src/addon/manager_ui.cpp

```diff
@@ -38,6 +39,7 @@ static lg::log_domain log_filesystem("filesystem");
 static lg::log_domain log_addons_client("addons-client");
 
 #define ERR_CFG LOG_STREAM(err,   log_config)
+#define INFO_CFG LOG_STREAM(info,   log_config)
```

Comment:
Either make the `log_config` line up with the line above, or don't add the extra spaces.

Reply:
Done.